Community-Aktivitäten/README.md

```diff
@@ -165,6 +165,46 @@ Dauer: 2 Stunden (bei Bedarf auch gerne mehr)
 
 Ausstattung: großer Screen/Beamer, Internet
 
+## Building your career: Ask anything
+:us: :de:
+
+### Collaborators
```

Comment:
Please note that the collaborators section will not end up in the final text. You can keep it here if you like, but what we actually need from you in any case is a *marked* person which is the one we will contact for organizational reasons (and which is the sole one getting a free ticket). Can you please set that mark? Thanks.

Reply:
Curiously the name and links are similarly added to other activities in the final document. Sadly that part was not so clear on the template. On the topic of tickets, As all the collaborators are part of the Content Committee Programs I am fairly certain they have tickets

Reply:
Anyways, for each community activity we need to know exactly one single person who is "officially in charge" (e. g. that gets called when we have to shift to another room etc.). So will that be you in all cases? Then I will note that.

Reply:
Well for the German version Hendrik is very keen and I would be for the English version! I still believe that adding us a group has advantages!